### DISCUSSION TITLE
Strange filter behaviour

### DISCUSSION THREAD

**Discussion by synapse** [pocketbase/pocketbase#1112] [Q&A] [answered]
I am trying to filter by multiple fields and it seams that either it is not considering some filter params or it's treating them as an `or`

`/api/collections/theCollection/records?filter=dateField=""&&usersField~"user-id"&&booleanField=false`
☝️ this works but returns all records. Doing 1 filter one by one also works - but the result is not the same.

if I add parenthesis (just like in the API example) returns an error
`/api/collections/theCollection/records?filter=(dateField=""&&usersField~"user-id"&&booleanField=false)`

``` json
{
    "code": 400,
    "message": "Invalid filter parameters.",
    "data": {}
}
```

N.B.: also trying using single quotes and space, the result is the same.

Any ideas of what am I doing wrong?

**Comment by ganigeorgiev** [comment, marked as answer]
If you are not using the JS or Dart SDK, make sure that the above filter query param value is properly url encoded. The last url of your examples should look something like this:

```
/api/collections/theCollection/records?filter=%28dateField%3D%22%22%26%26usersField~%22user-id%22%26%26booleanField%3Dfalse%29
```

**Reply by synapse** [reply]
That worked, thank you!
